src/awx_plugins/credentials/aim.py

```diff
@@ -1,6 +1,6 @@
 from urllib.parse import quote, urlencode, urljoin
 
-import requests
+import requests  # type: ignore[import-not-found]
```

Comment:
This should be added to https://github.com/ansible/awx-plugins/blob/472fb6f/.pre-commit-config.yaml#L189-L191 instead of ignoring.